qa/acceptance/sys/ubuntu/user_bootstrap.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+##
+# Installing rbenv
+##
+#git clone git://github.com/sstephenson/rbenv.git .rbenv
+#echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bash_profile
+#echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
+
+#git clone git://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
+#echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.bash_profile
+#source ~/.bash_profile
+
+
+##
+# install logstash from source code
+##
+#git clone https://github.com/elastic/logstash.git
+
+##
+# install jruby-1.7.24 and make it global
+##
+#rbenv install jruby-1.7.24
+#rbenv global  jruby-1.7.24
+
+#cd logstash
+#rake bootstrap
+#rake test:install-core
+
+cd
+wget https://download.elastic.co/logstash/logstash/packages/debian/logstash_2.3.1-1_all.deb
```

Comment:
What if we keep the scripts the same, they always install a package from the shared directory.

But we add code that is run (could be a rake task) before that to make sure the package is available in the shared directory. So the script could either download the package or ask logstash to build a new one.